gather.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+// Gatherer returns the metric gatherer
+func Gatherer() prometheus.Gatherer {
```

Comment:
`Gatherer()` is useful when we want to include the default registry in debug mode to export more metrics. Prometheus has a concept of `Gatherers` which is an `[]Gatherer` and merges gatherers in `Gather()`. This would all you to combine the two registries in debug mode to export them all in `/metrics`. 

The use case I have for `Gather()` is to allow sending metrics between workers & managers for aggregating container stats. Yes, you could always have the managers hit `/metrics`; however, the majority of those metrics potentially aren't valuable (containers not in services and daemon metrics). This would allow the worker to filter metrics to only tasks & send those over the wire saving bytes. Then you can aggregate all these in the manager